tuna/go_fish.py

```diff
@@ -60,20 +66,25 @@ def parse_args():
   return vars(args)
 
 
-def main():
+def main() -> bool:
   """Main function to start Tuna"""
   LOGGER.info(sys.argv)
   LOGGER.info(len(sys.argv))
+
+  args: Dict[str, Any]
   args = parse_args()
 
   #case no yaml file
+  library: Union[Example, MIOpen]
+  yaml_files: List[str]
   library = get_library(args)
   yaml_files = [args['yaml']]
 
   #case with yaml file
   if args['yaml']:
     yaml_files = parse_yaml(args['yaml'], args['lib'])
 
+  worker_lst: list
```

Comment:
what is this type? i dont see this imported anywhere, i see 'List' but not list

Reply:
list is a built in generic type( python 3.9) . List is defined in typing lib, which I hav not used here.